ui/Screen/DesignSystemGuide.svelte

```diff
@@ -212,6 +212,14 @@
     <Icon.SearchSmall />
     <Icon.Source />
     <Icon.Spinner />
+    <Icon.TxState />
+    <Icon.TxState progress={1} />
+    <Icon.TxState progress={2} />
+    <Icon.TxState progress={3} />
+    <Icon.TxState progress={4} />
+    <Icon.TxState progress={5} />
+    <Icon.TxState state="success" />
+    <Icon.TxState state="error" />
```

Comment:
Maybe it would be nice to distinguish the icons a little by putting them into their own section.
The spinner icon could also be separate.